build.sbt

```diff
@@ -914,6 +915,7 @@ lazy val `scio-extra`: Project = project
       "org.scalanlp" %% "breeze" % breezeVersion,
       "org.slf4j" % "slf4j-api" % slf4jVersion,
       "org.typelevel" %% "algebra" % algebraVersion,
+      "com.spotify" % "voyager" % voyagerVersion,
```

Comment:
IMHO we should revisit the `scio-extra` module to:
- create more sub modules
- mark deps as `provided`

otherwise user depending on `scio-extra` for the json feature only will pull tons of unwanted dependencies.

Reply:
I had a chat with Kellend about this and I brought up the same concern but I think that might be a larger than this PR concern? Or would you suggest having its own sub module itself?

Reply:
we'll do this in a later step, probably for 0.14